### ISSUE TITLE
Unhandled Exception EDD 5.1.0 beta portable.

### ISSUE THREAD

**Issue by Gimi1967** [EDDiscovery/EDDiscovery#573]
I get the following Unhandled Exception when I try to start EDD 5.1.0 beta portable.

```
 ---------------------------
Unhandled Exception
---------------------------
There was an unhandled exception.
Please report this at https://github.com/EDDiscovery/EDDiscovery/issues and attach C:\Programs\EDDiscovery\Data\Log\Trace_20161123211616.log
Exception: System.IO.FileLoadException: Could not load file or assembly 'OpenTK, Version=1.1.0.0, Culture=neutral, PublicKeyToken=bad199fe84eb3df4' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

File name: 'OpenTK, Version=1.1.0.0, Culture=neutral, PublicKeyToken=bad199fe84eb3df4'
   at EDDiscovery2.FormMap.InitializeComponent()
   at EDDiscovery2.FormMap..ctor()
   at EDDiscovery2._3DMap.MapManager..ctor(Boolean nowindowreposition, TravelHistoryControl tc)
   at EDDiscovery.EDDiscoveryForm..ctor()
   at EDDiscovery.Program.Main()

WRN: Assembly binding logging is turned OFF.

To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

This application must now close
---------------------------
OK   
---------------------------
```

Tracelog:

```
Native library pre-loader is trying to load native SQLite library "C:\Programs\EDDiscovery\x64\SQLite.Interop.dll"...
SQLite version 3.14.2
Theme ID 10037
First chance exception: Could not load file or assembly 'OpenTK, Version=1.1.0.0, Culture=neutral, PublicKeyToken=bad199fe84eb3df4' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at EDDiscovery2.FormMap.InitializeComponent()
   at EDDiscovery2.FormMap..ctor()
   at EDDiscovery2._3DMap.MapManager..ctor(Boolean nowindowreposition, TravelHistoryControl tc)
   at EDDiscovery.EDDiscoveryForm..ctor()
   at EDDiscovery.Program.Main()


==== UNHANDLED EXCEPTION ====
System.IO.FileLoadException: Could not load file or assembly 'OpenTK, Version=1.1.0.0, Culture=neutral, PublicKeyToken=bad199fe84eb3df4' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
File name: 'OpenTK, Version=1.1.0.0, Culture=neutral, PublicKeyToken=bad199fe84eb3df4'
   at EDDiscovery2.FormMap.InitializeComponent()
   at EDDiscovery2.FormMap..ctor()
   at EDDiscovery2._3DMap.MapManager..ctor(Boolean nowindowreposition, TravelHistoryControl tc)
   at EDDiscovery.EDDiscoveryForm..ctor()
   at EDDiscovery.Program.Main()

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

==== cut ====
```

**Comment by klightspeed** [comment]
Please try downloading https://raw.githubusercontent.com/klightspeed/EDDiscovery/d43e373e535bc7dfb4516a7bbf9039de3fc83650/EDDiscovery/App.Portable.config, renaming it to `EDDiscovery.exe.config`, and copying it into `C:\Programs\EDDiscovery`, replacing the existing config config file.

The portable config file was missing the OpenTK DLL binding redirect.

**Comment by Gimi1967** [comment]
That worked, thanks.

**Comment by finwen** [comment]
Closed woth EDD 5.1.1
